source/androidx.activity/activity-compose/Additions/Additions.cs

```diff
@@ -0,0 +1,4 @@
+﻿using System;
```

Comment:
I realize all the other projects in this repo also suffer from this, but adding empty `Additions.cs` and `.xml` transforms files only slow down the build process, as each "empty" file must be read from disk and parsed before the tooling knows it can ignore it.

Reply:
Should I make any change?